public/js/Lineplot.js

```diff
@@ -214,13 +201,14 @@ Lineplot.prototype.keypressUpdate = function(keyCode) {
     var self = this;
 
     if(self.activeFeature) {
-        if(keyCode === RIGHTARROW && self.activeFeature < this.featureLength) {
+        if(keyCode === RIGHTARROW &&
+            self.activeFeature < this.featureVector.length) {
```

Comment:
Is this 0- or 1-indexed? If 0, won't this fail when we get to the rightmost feature and try to increment it?